bemani/frontend/museca/endpoints.py

```diff
@@ -139,6 +139,7 @@ def viewrecords(userid: UserID) -> Response:
     if info is None:
         abort(404)
     versions = {version: name for (game, version, name) in frontend.all_games()}
+    versions[10002] = 'MÚSECA PLUS'  # Hard code this version name for modpack songs
```

Comment:
This seems a bit suspect. I think I see what you're doing here, and it relates to the fact that MUSECA+ is not actually an onminix, but a fan continuation with new original songs. IIDX omnimix still made the assumption that every song would be found in SOME previous version, so there didn't need to be a separate category for it. This kinda makes the argument that this isn't really an omnimix so much as it is an expansion or a new mix entirely.

Granted, since its isolated to a dead game, and in the game's specific code, I think it's probably fine. However, that's the point of the `all_games()` method. I think you should be adding a hardcoded entry (and a comment as to why) here instead: https://github.com/DragonMinded/bemaniutils/blob/trunk/bemani/frontend/museca/museca.py#L25

I don't know why it was done in the spot it was for Pop'n.